index.html

```diff
@@ -34,5 +34,131 @@
       </li>
     </ul>
     </div>
+    <div id="recent">
+      <div id="title"><p>My Recent Works</p></div>
+      <span id="line"><hr/></span>
+    <div id="frame">
+        <div id="caption_img"> <img src="./Image/Img Placeholder.png"/></div>
+        <div id="content">
+        <div id="stories"><p>Multi-Post Stories</p></div>
+        <div id="summary">A daily selection of privately personalized reads, no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500's, when an unknown printer took a standard dummy text.</div>
+    <div id="bar">
+        <div id="css">css</div>
+        <div class=""="html">html</div>
+        <div class=""="boot">bootstrap</div>
+        <div class=""="ruby">Ruby</div>
+    </div>
+      <button id="button" type="button">See Project</button>
+      </div>
+  <div id="grid">   
```

Comment:
Good work implelmenting some of the project requirement but unfortunately you still need tp fix some issues before you proceed. While reviewing your work I notice your we page doesnt fit a mobile screen size and it has a white space. Your web app should fit the exact phone size screen please follow the figma design properly and pay more attention to detail. And Kindly make your screen size responsive 
This is your design:
![Screenshot from 2022-07-02 16-10-22](https://user-images.githubusercontent.com/79506105/177000436-a6e91b02-a816-4490-9d41-041ce8fd9b22.png)


Your design should be like this:
![Screenshot from 2022-07-02 16-12-13](https://user-images.githubusercontent.com/79506105/177000569-5ae1d841-4dab-40e4-af4d-bd8e1ac7a168.png)